### ISSUE TITLE
Error: Wrong number of fields

### ISSUE THREAD

**Issue by matrixtushar** [grafana/grafana-csv-datasource#53]
Hi,

I have a CSV that only has two fields and two rows. I have placed this csv on a public git repo and confirmed that I am able to access it manually without a problem.

I added the url and tested, which resulted in success. However, in the dashboard it keeps saying 
record on line 74: wrong number of fields
I also tried your example csv from John Hopkins data. That also results in the same error.

**Comment by jaknawago** [comment]
Hello,
I have the same issue. Double checked the csv if its ok. Tried to shrink its size to just couple of row, but still the same error:`record on line 74: wrong number of fields`. Even that there in no line 74 in a file.

**Comment by marcusolsson** [comment]
Thank you for reporting this! Could you provide an example of a CSV produces this error?

**Comment by marcusolsson** [comment]
The README listed the wrong path (it should point to the raw content). I've updated the docs in response to [this issue](https://github.com/marcusolsson/grafana-csv-datasource/issues/64).

Also related, some servers don't seem to like the default Accept header sent by the plugin. I've removed it as mentioned in [this issue](https://github.com/marcusolsson/grafana-csv-datasource/issues/56).

I hope that will be enough to close this issue. Let me know if you're still having issues.
